### ISSUE TITLE
augmentation of teacher model config 

### ISSUE THREAD

**Issue by ywdong** [microsoft/Cream#126]
Hi，I have the question about the config of teacher model，with the config：configs/teacher/clip_vit_large_patch14_22k.yaml
`MODEL:                                                                                                                  
  TYPE: clip_vit_large14_224                                                                                            
                                                                                                                        
TRAIN:                                                                                                                  
  EPOCHS: 90                                                                                                            
                                                                                                                        
DATA:                                                                                                                   
  MEAN_AND_STD_TYPE: clip                                                                                               
  DATASET: imagenet22k                                                                                                  
                                                                                                                        
AUG:                                                                                                                    
  MIXUP: 0.0                                                                                                            
  CUTMIX: 0.0 `
  Why the augmentation is 0.0? Will the predicted logits obtained in each round be the same？
  Thx for your quick response~

**Comment by wkcn** [comment]
Thanks for your attention to our work!

We disable Mixup and Cutmix for the distillation on ImageNet-22k, therefore `MIXUP` and `CUTMIX` are set to 0.0.

However, the other augmentations like color jitter, random horizontal flipping are reserved.

Related Code: https://github.com/microsoft/Cream/blob/main/TinyViT/data/build.py#L172
